ui/package.json

```diff
@@ -17,9 +17,14 @@
 		"react-tap-event-plugin": "3.0.2"
 	},
 	"devDependencies": {
+		"chai": "^4.1.2",
```

Comment:
i've not done any frontend testing before but since this is a create react app it comes with jest which i understood was reasonably standard. do you have any experience with jest? does this stack offer any advantages that warrant the extra dependencies?

Reply:
Oh sure man, yea you're right jest does all the same stuff.